mix.exs

```diff
@@ -50,7 +50,7 @@ defmodule App.MixProject do
       {:gettext, "~> 0.11"},
       {:jason, "~> 1.0"},
       {:plug_cowboy, "~> 2.0"},
-      {:elixir_auth_google, "~> 0.1.0" },
+      {:elixir_auth_google, "~> 1.0.1" },
```

Comment:
version `1.0.2` has the changes made in PR https://github.com/dwyl/elixir-auth-google/pull/24